README.md

```diff
@@ -59,7 +59,7 @@ The following tools help make easier to work with sample code.
 
 ## Authors
```

Comment:
Kindly work on these changes:
- Implement a function for the "Clear all completed" button (`use filter()` method).
- When i try to remove a task from the delete icon doesn't works well. i hit multiple time at this button and i had this error displayed:

![Screen Shot 2021-12-04 at 9 27 36](https://user-images.githubusercontent.com/22258274/144715093-a03b8f66-f72f-4f1f-8786-a43601136360.png)

Reply:
Done - thanks for the assistance, I have changed some things on the list so it can work